cli/determined_cli/__init__.py

```diff
@@ -1 +1,7 @@
-from .__version__ import __version__
```

Comment:
I have a very, very small preference for leaving the `__version__` here so that when we remove this dummy package then nobody will still have code that looks for a `det.cli.__version__` symbol

Reply:
fair point, done.